docs/9.0/api/manualRowMove.md

```diff
@@ -188,12 +188,12 @@ _manualRowMove.moveRows(rows, finalIndex) ⇒ boolean_
 
 Moves a multiple rows.
 
-**Emits**: [`Hooks#event:beforeRowMove`](./hooks/#beforeRowMove), [`Hooks#event:afterRowMove`](./hooks/#afterRowMove)  
+**Emits**: [`Hooks#event:beforeRowMove`](@/api/pluginHooks.md#beforerowmove), [`Hooks#event:afterRowMove`](@/api/pluginHooks.md#afterrowmove)  
 
 | Param | Type | Description |
 | --- | --- | --- |
 | rows | `Array` | Array of visual row indexes to be moved. |
-| finalIndex | `number` | Visual row index, being a start index for the moved rows. Points to where the elements will be placed after the moving action. To check the visualization of the final index, please take a look at [documentation](@/guides/rows/row-moving.md#drag-and-move-actions-of-manualrowmove-plugin). |
+| finalIndex | `number` | Visual row index, being a start index for the moved rows. Points to where the elements will be placed after the moving action. To check the visualization of the final index, please take a look at [documentation](@/guides/rows/row-moving.md#dragandmoveactionsofmanualrowmoveplugin). |
```

Comment:
`#drag-and-move-actions-of-manualrowmove-plugin`

Reply:
#8440